src/apiCalls.js

```diff
@@ -9,20 +9,34 @@ export const getMovieData = (movieUrl) => {
        return fetch(characterUrl)
          .then(response => response.json())
          .then(characterObj => {
-            const { name, homeworld, species, films} = characterObj;
-            console.log(name)
-               return getHomeworld(homeworld)
-               .then(data => ({data, name, species, films}));
-            // Promise.all(characterInfo);
+            const { films, homeworld, species } = characterObj;
+            let promise1 = getFilms(films).then(filmTitle => filmTitle);
+            let promise2 = getHomeworld(homeworld).then(worldObj => ({homeWorldName: worldObj.name, homeWorldPop: worldObj.population}));
+            let promise3 = getSpecies(species).then(speciesObj => speciesObj.name)
+            return Promise.all([promise1, promise2, promise3]).then(data => ({name: characterObj.name, species: data[2], homeworldName: data[1].homeWorldName, homeworldPop: data[1].homeWorldPop, appearsIn: data[0], favorited: false}))
```

Comment:
This looks great!! Would be great to look over it together tomorrow so you can explain to me how this magic happened.